pygmt/src/grd2cpt.py

```diff
@@ -159,6 +161,20 @@ def grd2cpt(grid, **kwargs):
         range. Note that ``cyclic=True`` cannot be set together with
         ``categorical=True``.
     {verbose}
+
+    Example
+    -------
+    >>> import pygmt
+    >>> # load the 30 arc minute grid with "gridline" registration
+    >>> grid = pygmt.datasets.load_earth_relief("30m", registration="gridline")
+    >>> # Create a plot
+    >>> fig = pygmt.Figure()
+    >>> # create a CPT from the grid object with grd2cpt
+    >>> pygmt.grd2cpt(grid=grid)
+    >>> # Plot the grid object, the CPT will be automatically used
+    >>> fig.grdimage(grid=grid)
+    >>> # Show the plot
+    >>> fig.show()
```

Comment:
I feel it would be nice to start the comments consistently with either upper- or lower-case letters. I'm also fine with upper-case letters.
```suggestion
    Example
    -------
    >>> import pygmt
    >>> # load the 30 arc minute grid with "gridline" registration
    >>> grid = pygmt.datasets.load_earth_relief("30m", registration="gridline")
    >>> # create a plot
    >>> fig = pygmt.Figure()
    >>> # create a CPT from the grid object with grd2cpt
    >>> pygmt.grd2cpt(grid=grid)
    >>> # plot the grid object, the CPT will be automatically used
    >>> fig.grdimage(grid=grid)
    >>> # show the plot
    >>> fig.show()
```